manifests/config_map.yaml

```diff
@@ -3,6 +3,4 @@ kind: ConfigMap
 metadata:
   name: vm-console-proxy
 data:
-  tls-profile-v1alpha1.yaml: |
-    type: Intermediate
-    intermediate: {}
+  tls-profile-v1.yaml: ""
```

Comment:
Example not required anymore?

Reply:
I didn't want to specify any configuration, but at the same time I wanted to keep the file in ConfigMap, so it is clear what filename should be used.

IMHO it is better to use empty file, which will result in default TLS configuration, than to specify some configuration here. Otherwise we would need to keep it updated.